book_source/06_reference/10_editing_this_book.Rmd

```diff
@@ -0,0 +1,62 @@
+# Editing this book {#bookediting}
```

Comment:
```suggestion
# Editing This Book {#bookediting}
```

Reply:
...Though now that I check, it looks like we have more chapter titles in sentence case than title case, so maybe this is correct already

Reply:
Because we tend to use sentence case more often than title case, I'm going to leave this as is.